Assets/Python/GreatPeople.py

```diff
@@ -3504,14 +3518,18 @@ def setup():
 		"Clemens Brenninkmeijer", # 19th
 		"August Kessler", # 19th
 		iGlobal,
+		"Anton Philips", # 20th
```

Comment:
There is also a Frits Philips in the Great Engineers list. (Which is the son of Anton) I think it's better to only use one of them. (Anton in probably better then Frits) And I think they fit the role of Great Engineer more than Great Merchant.

Reply:
Thanks for verifying the Dutch GP! I'll change that.